### ISSUE TITLE
Improve accessibility of oral arguments audio player

### ISSUE THREAD

**Issue by elisa-a-v** [freelawproject/courtlistener#5589]
The current implementation of the audio player isn't super accessible, in particular:
- The sliders are tiny and they don't support keyboard interaction, so they're only really usable with a fine pointer. This renders them basically useless in touchscreens, and even with a mouse it can be hard to get the exact value you want.
- Semantics needs to be improved:
	- Some elements could be replaced with semantic HTML, or at least have proper ARIA roles. [Here](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements#forms) is a list of all semantic elements available, the ones under Form are probably some of the most appropriate for the interactive elements.
	- Some elements need relevant ARIA attributes, especially the interactive ones.
	- For example, sliders should probably have `role="slider"`, `aria-valuemin`, `aria-valuemax`, `aria-valuenow`. See [ARIA docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) for further reference.
- All interactive elements [must be focusable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Keyboard#interactive_elements_must_be_focusable) and [must have clear focus styling](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Keyboard#focusable_element_must_have_focus_styling). Several interactive elements aren't focusable, and the focus ring is barely visible.
- The grey colors don't meet WCAG contrast standards.

[WCAG quick reference guide](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1)

**Comment by mlissner** [comment]
Thanks for filing this. It's definitely not a priority to do any of this in the current design, but we can look at it a bit as we're working on the new one. 

I don't think @s-taube has made issues for that yet, but I'll this as a child to the redesign epic so it doesn't get lost, and when Stephanie gets to the oral argument pages, we can revisit this.

**Comment by s-taube** [comment]
After the API help pages, the next pages we want Magnet's developer to work on are: judges pages, and oral arguments. So, I'm queueing up design work on those two pages to make sure they're ready for development in the near future.

On that note, how would you prioritize this issue, given that there's an ever-expanding backlog of design work? Should this be addressed before Magnet's developer gets to the page? @mlissner cc @elisa-a-v

**Comment by elisa-a-v** [comment]
Thank you @s-taube for bringing this up now!

> Should this be addressed before Magnet's developer gets to the page?

Yes, these are all things that should be considered in the new audio player design itself. If we don't address this beforehand, we'll probably have to redo things, or we may end up with an inaccessible audio player.

This issue was opened thinking more about tweaking the current player, but if we'll just address this in the new design, this is _not a separate task_ than designing+implementing the new audio player, it's more like added requirements to meet accessibility standards.

We have a volunteer who [wants to help](https://github.com/freelawproject/courtlistener/discussions/5701) with the audio player though, if Magnet's dev will take it over we should probably let them know. They haven't started as far as I know.

#### For design

From the bullet points listed above, the first and last items are relevant **for design**: we need to make sure all control sizes are big enough, and that color contrast meets WCAG standards.

These are just some basic considerations, but W3C WAI provides more [tips on designing for web accessibility](https://www.w3.org/WAI/tips/designing/) to help meet WCAG requirements. Each tip links to the relevant WCAG success criteria.

#### For development

Whoever actually **develops** the new audio player should also consider accessibility as described in the issue above. For further reference, [here](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Accessibility/Multimedia)'s an MDN article about accessible multimedia in general, and [another one](https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-seek/) about seek sliders in particular.

**Comment by s-taube** [comment]
@elisa-a-v checking out the [figma ](https://www.figma.com/design/OEDp7KwNSpJDyJRiKrJmm4/Court-Listener?node-id=7535-49409&t=O55sLTz6pOwWNC5i-0)- it looks like the first and last points you mentioned are still issues.

<img width="612" height="318" alt="Image" src="https://github.com/user-attachments/assets/f0c2a330-ef79-4994-83d7-3061b91d9527" />

I'll add a card to the top of the backlog to update this design. For development, I'll leave it to you two to decide who gets assigned.

**Comment by s-taube** [comment]
The "play" button got smaller, too...

**Comment by elisa-a-v** [comment]
> it looks like the first and last points you mentioned are still issues.

Yeah I think you're right :(

> I'll add a card to the top of the backlog to update this design.

Nice, thank you!
